### PR TITLE
Added like functionality

### PR DESCRIPTION
**Pull request by aiman-ansari**
- User can like videos only when logged-in.
- User can remove videos from liked videos

### CODE REVIEW

**Review by mansipadgelwar — APPROVED** [review]
Looks good.

File: src/Context/LikeContext.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+const LikeContext = createContext();
+const useLikes = () => useContext(LikeContext)
+
+const LikeContextProvider = ({children}) =>{
+    const [likes, setLikes] = useState([])
+    useEffect(()=>{
+        getLike()
+    },[])
```
Comment (rahulyadav139) [inline src/Context/LikeContext.js:13]: suggestion: you should include getLike in the dependency array. You can react memoization to do that.

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: Codes are readable. but formatting is bit not in  correct order. You can use prettier extension to fix that. Other than great work.

**Review by devAmlan — APPROVED** [review]
Codes looks good, Use var color in background color , and use rem .

**Pull request merged**